src/routes/console/project-[project]/updateVariables.svelte

```diff
@@ -138,9 +138,11 @@
             selectedVar = null;
             showPromoteModal = false;
 
-            await invalidate(Dependencies.FUNCTION);
-            await invalidate(Dependencies.VARIABLES);
-            await invalidate(Dependencies.PROJECT_VARIABLES);
+            await Promise.all([
+                await invalidate(Dependencies.FUNCTION),
+                await invalidate(Dependencies.VARIABLES),
+                await invalidate(Dependencies.PROJECT_VARIABLES)
```

Comment:
Do we really need the extra await in all of these?

```suggestion
               invalidate(Dependencies.FUNCTION),
               invalidate(Dependencies.VARIABLES),
               invalidate(Dependencies.PROJECT_VARIABLES)
```